### ISSUE TITLE
parse error on sequential casting

### ISSUE THREAD

**Issue by taichi-ishitani** [dalance/svlint#61]
I got an error during parsing source file including sequential casting.
You can reproduce this error by using this code.

```verilog
module top;
  int a = 17'(1+2)'(1+2);
endmodule
```

```
Error: parse error
   --> test.sv:2:19
  |
2 |   int a = 17'(1+2)'(1+2);
  |                   ^
```

I think this should be accepted according to following rules.

* constant_cast ::= casting_type ' ( constant_expression )
* casting_type ::= simple_type | constant_primary | signing | string | const
* constant_primary ::= constant_cast

In addition, commercial simulators, e.g. VCS, Xcelium, allows the above code.
 
* svlint version
    * 0.4.17

**Comment by DaveMcEwan** [comment]
This issue should be raised on [sv-parser](https://github.com/dalance/sv-parser) instead of svlint.
@dalance Would you close this issue?
